src/sql/base/parts/editableDropdown/browser/dropdown.ts

```diff
@@ -344,8 +344,8 @@ export class Dropdown extends Disposable implements IListVirtualDelegate<string>
 	}
 
 	public set value(val: string) {
+		this._input.value = val;
```

Comment:
Why are you moving this? It seems like the underlying inputbox is also doing this check so I guess it should be fine, but is there a reason you needed to do this in the first place?

Reply:
There was a bug in editable dropdown that does not show the currently selected option when we accidentally type over a selected option and then select the same option again.  This line of code fixes that change.

Reply:
Is this a current issue? In the future it would be good to submit fixes like this in a separate PR. At the very least make sure you explicitly call out that you're fixing an issue and describe what the issue.

Reply:
Yes. It is an issue in the current state of the dropdown. I will avoid clubbing issue fixes like these in the future with feature PRs